keychain-environment.el

```diff
@@ -49,6 +49,29 @@
 
 ;;; Code:
 
+(defvar keychain-should-inherit
+  nil
+  "Should the keychain invocation should inherit.
+Can be either nil, local, any or t. See keychain manual.")
```

Comment:
There's something wrong with the first sentence. Also I think this should document what this does and what the various values mean without having to refer to the manpage.